ui/pages/routes/routes.component.js

```diff
@@ -448,7 +452,8 @@ export default class Routes extends Component {
     });
     const shouldShowNetworkInfo =
       isUnlocked &&
-      currentChainId &&
+      account &&
+      isCurrentChainCompatibleWithAccount(currentChainId, account) &&
```

Comment:
Maybe we should also check for `currentChainId` (as before), like:

```suggestion
      (currentChainid && isCurrentChainCompatibleWithAccount(currentChainId, account)) &&
```

Reply:
Alternatively, we could (should?) check it in `isCurrentChainCompatibleWithAccount`